develop/items/custom-tools.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Tools and Weapons
+description: Learn how to create your own tools and set their durability and mining levels.
+authors:
+  - IMB11
+---
+
+# Tools
+
+Tools are essential for survival and progression, allowing players to gather resources, construct buildings, and defend themselves.
+
+## Creating a tool material.
```

Comment:
Maybe mention the vanilla `ToolMaterials` class somewhere here, to give devs context in terms of values?